src/coreclr/md/compiler/custattr_emit.cpp

```diff
@@ -991,9 +991,9 @@ HRESULT RegMeta::_IsKnownCustomAttribute(        // S_OK, S_FALSE, or error.
 
     // Add to hash.
     sLookup.ca = ixCa;
-    pFound = m_caHash.Add(&sLookup);
-    IfNullGo(pFound);
-    *pFound = sLookup;
+    if (m_caHash.AddNoThrow(sLookup) != TRUE)
```

Comment:
```suggestion
    if (!m_caHash.AddNoThrow(sLookup))
```
Static analyzers like to complain about `!= TRUE`.